README.md

```diff
@@ -7,35 +7,41 @@ Printing since 1440.
+This repo is the development hub for the <a href="https://make.wordpress.org/core/2017/01/04/focus-tech-and-design-leads/">editor focus in core</a>. `Gutenberg` is the project name.
 
-Discover more about the project <a href="https://wordpress.org/gutenberg">here</a>.
+## Getting started
+- **Download:** If you want to use the latest release with your WordPress, <a href="https://wordpress.org/plugins/gutenberg/">download the latest release from the WordPress.org plugins repository</a>. 
+- **Discuss:** Conversations and discussions take place in #core-editor on <a href="https://make.wordpress.org/chat/">the core WordPress Slack</a>.
+- **Contribute:** Development happens here. Get started by reading the <a href="https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md">CONTRIBUTING</a> guidelines.
+- **Learn:** Discover more about the project <a href="https://wordpress.org/gutenberg">here</a>.
 
-Gutenberg is more than an editor. While the editor is the focus right now, the project will ultimately impact the entire publishing experience including customization (the next focus area).
+**Gutenberg is more than an editor.** While the project is currently focused on building the new editor for WordPress, it doesn't end there. This lays the groundwork for a new model for WordPress Core that will ultimately impact the entire publishing experience of the platform.
 
 ## Editing focus
 
-> The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery. — Matt Mullenweg
+> *The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery.*
+>
+> — Matt Mullenweg
 
-One thing that sets WordPress apart from other systems is that it allows you to create as rich a post layout as you can imagine -- but only if you know HTML and CSS and build your own custom theme. By thinking of the editor as a tool to let you write rich posts and create beautiful layouts, we can transform WordPress into something users _love_, as opposed something they pick it because it's what everyone else uses.
+One thing that sets WordPress apart from other systems is that it allows you to create a post layout as rich as you can imagine—but only if you know HTML and CSS and build your own custom theme. By thinking of the editor as a tool that can let you both write rich posts and create beautiful layouts, we can transform WordPress into something users _love_, as opposed something they pick because it happens to be what everyone else uses.
```

Comment:
It was in the original text, but 'as opposed something' seems like a mistake. Shouldn't it be 'as opposed to something', or 'instead of something'.

Reply:
Hello @talldan, thanks for the copy review ping. Here's a suggestion to add a couple of words that seem to be missing and streamline the text just slightly. 

```suggestion
One thing that sets WordPress apart is that it allows you to create a post layout that's as rich as you can imagine—but only if you know HTML and CSS and can build your own custom theme. By thinking of the editor as a tool that allows you to write rich posts **and** create beautiful layouts, we can transform WordPress into something users _love_, as opposed to something they choose because it happens to be what everyone else uses.
```